src/components/status.jsx

```diff
@@ -94,6 +94,7 @@ const visibilityText = {
   unlisted: msg`Unlisted`,
   private: msg`Followers only`,
   direct: msg`Private mention`,
+  local: msg`Local instance`,
```

Comment:
I'm wondering is this the copy that Pleroma/Akkoma uses?

Reply:
Pleroma does not expose that in its frontend at all.

Akkoma uses "Local - do not federate this post" (and "Direct - post to mentioned users only" instead of "Private mention").

Reply:
I've read this https://docs.pleroma.social/frontend/user_guide/posting_reading_basic_functions/ and it doesn't seem to mention local-only posting. Is it some feature from (yet) another Pleroma fork?

I'm fine with this "Local instance" copy if it is within user's expectations.

Reply:
Pleroma's backend supports local-only posting, but (like with many other Pleroma features, such as list-only posting or scrobbling) does not expose it in the frontend for some reason.

Akkoma added a button to the frontend to toggle it on and off.

Reply:
I did a little search and found hometown's doc where they use the copy "Local-only" 🤔 https://github.com/hometown-fork/hometown/wiki/Local-only-posting

If this is added, I think it's better that it's not placed at the bottom of "Private Mention" (not this line, but on `compose.jsx`). Maybe below "Unlisted"? As in it's sorted by uh… how "far" a post propagates…? 🤔

Reply:
Wow this is useful. Any idea how this table would look like if we add "Local" from Hometown/Misskey/*key forks one day? 🙈

I'm fine with this sort order (below "Public") and change the copy to "Local". Reasons + notes:

- I'm trying to move away from the word "instance" as I think non-technical users are not familiar with it. "server" seems to be a better word. There are many "instance" copies on Phanpy that's not changed yet for now.
- This visibility select menu needs a better UI. It should be "label" + "description" under it (like Mastodon web or other clients). Having just "label" doesn't describe what they do. I haven't look into this yet.
- If "Local" is used, it'll be matched with "Local" copy from the local timeline, which should be same contextually?

Reply:
I don't have an account on Misskey to confirm it's available there (and I'm not even sure it's compatible with the Mastodon API in first place), but I had a Firefish instance, and I've had a test account on an Iceshrimp one, and there "Local" is an on-off toggle separate from visibility:

## On Firefish and Iceshrimp

| Visibility | Direct messages | Home timeline | Public profile | Local timeline | Federated timeline |
|-|-|-|-|-|-|
| Public | :x: | :green_circle: | :green_circle: | :green_circle: | :green_circle: |
| Local Public | :x: | :large_blue_diamond: | :large_blue_diamond: | :large_blue_diamond: | :large_blue_diamond: |
| Unlisted | :x: | :yellow_square: | :green_circle: | :x: | :x: | :x: |
| Local Unlisted | :x: | :large_orange_diamond: | :large_blue_diamond: | :x: | :x: | :x: |
| Followers | :x: | :yellow_square: | :yellow_square: | :x: | :x: |
| Local Followers | :x: | :large_orange_diamond: | :large_orange_diamond: | :x: | :x: |
| Direct | :white_heart: | :white_heart: | :x: | :x: | :x: |
| Local Direct | :orange_heart: | :orange_heart: | :x: | :x: | :x: |

| Emoji | Meaning |
|-|-|
| :green_circle: | Anybody can see |
| :large_blue_diamond: | ***Anybody can see if they look on the server the post was made, even without an account***, but not through federation (different from Pleroma/Akkoma!) |
| :yellow_square: | Followers and mentioned can see |
| :large_orange_diamond: | Any followers of the poster or mentioned in the post ***on the same server of the poster*** can see |
| :white_heart: | Mentioned can see |
| :orange_heart: | Mentioned ***on the same server of the poster*** can see |
| :x: | Not visible |